src/postgres/commands/createPostgresServer/steps/PostgresServerNameStep.ts

```diff
@@ -42,6 +42,8 @@ async function validatePostgresServerName(name: string, client: PostgreSQLManage
         return localize('serverNameLengthCheck', 'The name must be between {0} and {1} characters.', min, max);
     } else if (!(/^[a-z0-9-]+$/).test(name)) {
         return localize('serverNameCharacterCheck', 'Server name must only contain lowercase letters, numbers, and hyphens.');
+    } else if ((/^[0-9]+$/).test(name)) {
```

Comment:
I think the issue was talking about the admin username, not the server name

Reply:
That's true, created another PR to fix username. I noticed that serverName seems to have a similar problem with validation.

Reply:
So "12345" is invalid but "123-45" is valid? That seems like a bug, not intentional design. Did you find any docs on this? If not, I think I lean towards "wont-fix" and calling this Azure's problem

Reply:
It is odd. I didn't find any docs but when I tried to create using only numbers on the portal, it is not allowing. 
![image](https://user-images.githubusercontent.com/17537419/92643964-7a0edf80-f297-11ea-9b68-d051178fd394.png)
![image](https://user-images.githubusercontent.com/17537419/92644032-8dba4600-f297-11ea-854b-396e7da02cb6.png)

Reply:
Actually, same thing goes for username. The invalid case with just numbers is not mentioned on the portal, but it doesn't allow to create with it.

Reply:
Right, but the portal's message doesn't saying anything about "it can't be all numbers", so I think it's a bug in the portal.

I just tried creating a server from our extension with all numbers and it worked.

Reply:
I agree, it must be in the portal, creating through server works.